index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
+			const retryAfter = options !== undefined &&
+				['number', 'string'].includes(typeof options.retryAfter) ?
+				Number(options.retryAfter) :
+				2000;
```

Comment:
2 seconds might be quite fast for some users. Would 5 seconds be better?

Reply:
You might be right, I'm not really sure. It's a kill command after all - 2 seconds seems pretty long for that to finish imo. But I changed it: af15667e3d995374b2f22a8b8fc8bbf7d8418152

Reply:
Closing servers sometimes require network round-trips, i.e. can easily take few seconds.